tutorials/export/one-click_deploy.rst

```diff
@@ -117,3 +119,43 @@ To resolve this:
 - On Linux, you may be missing the required
   `udev rules <https://github.com/M0Rf30/android-udev-rules>`__
   for your device to be recognized.
+
+.. _doc_one-click_deploy_troubleshooting_web:
+
+Web
+^^^
+
+By default, the web server started by the editor is only accessible from
+``localhost``. This means the web server can't be reached by other devices on
+the local network or the Internet (if port forwarding is set up on the router).
+This is done for security reasons, as you may not want other devices to be able
+to access the exported project while you're testing it. Binding to ``localhost``
+also prevents a firewall popup from appearing when you use one-click deploy for
+the web platform.
+
+To make the local web server accessible over the local network, you'll need to
+change the **Export > Web > HTTP Host** editor setting to ``0.0.0.0``. You will
+also need to enable **Export > Web > Use TLS** as SharedArrayBuffer requires the
+use of a secure connection to work, *unless* connecting to ``localhost``.
+However, since other clients will be connecting to a remote device, the use of
+TLS is absolutely required here.
+
+To make the local web server accessible over the Internet, you'll also need to
+forward the **Export > Web > HTTP Port** port specified in the Editor Settings
+(``8060`` by default) in TCP on your router. This is usually done by accessing
+your router's web interface then adding a NAT rule for the port in question. For
+IPv6 connections, you should allow the port in the router's IPv6 firewall
+instead. Like for local network devices, you will also need to enable **Export >
+Web > Use TLS**.
+
+.. note::
+
+    When **Use TLS** is enabled, you will get a warning from your web browser as
+    Godot will use a temporary self-signed certificate. You can safely ignore it
+    and bypass the warning by clicking **Advanced** and then **Proceed to
+    (address)**.
+
+    If you have a SSL certificate that is trusted by browsers, you can specify
```

Comment:
```suggestion
    If you have a TLS certificate that is trusted by browsers, you can specify
```
I get that TLS certs are still commonly referred to as SSL certs, but I don't think we should do so here since Godot uses "TLS certificate" in the settings. I'd also be fine with calling it a "TLS/SSL Certificate"